jujugui/static/gui/src/app/components/profile/header/_header.scss

```diff
@@ -1,126 +1,13 @@
 .profile-header {
-  background-color: $white;
-  padding-top: 30px;
-  padding-bottom: 30px;
-  border-bottom: 1px solid $light-grey;
+  margin-bottom: 3rem;
```

Comment:
Can you comment that this is to compensate for the lack of a following `p-strip`